src/lib/pci_emul.c

```diff
@@ -572,7 +572,7 @@ pci_emul_alloc_pbar(struct pci_devinst *pdi, int idx, uint64_t hostbase,
 	limit = 0;
 
 	if ((size & (size - 1)) != 0)
-		size = 1UL << flsl((long) size); /* round up to a power of 2 */
+		size = 1UL << flsl((long)size); /* round up to a power of 2 */
```

Comment:
I think the signed-ness of the variable should not matter for `flsl` as it would just look at the bits. `long` is indeed 64bit so this should work...though i'm going to change this to `long long` which is a bit more correct, I think, at least on 32bit systems